build.sbt

```diff
@@ -1,29 +1,26 @@
 import ReleaseTransformations._
+import sbtversionpolicy.withsbtrelease.ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease
 
 name := "tags-thrift-schema"
 organization := "com.gu"
 
-scmInfo := Some(ScmInfo(url("https://github.com/guardian/tags-thrift-schema"), "scm:git@github.com:guardian/tags-thrift-schema"))
-homepage := scmInfo.value.map(_.browseUrl)
-developers := List(Developer(id = "guardian", name = "Guardian", email = null, url = url("https://github.com/guardian")))
-
-
 libraryDependencies ++= Seq(
   "org.apache.thrift" % "libthrift" % "0.13.0",
   "com.twitter" %% "scrooge-core" % "19.11.0"
 )
 
 // Might cross compile more scala versions here depending on who needs this!
-scalaVersion := "2.13.1"
-crossScalaVersions := Seq("2.11.12", "2.12.10", scalaVersion.value)
+scalaVersion := "2.13.11"
+crossScalaVersions := Seq("2.12.18", scalaVersion.value)
+scalacOptions := Seq(
+  "-release:11"
+)
 releaseCrossBuild := true
 
 resolvers += Resolver.jcenterRepo
 
```

Comment:
It's probably worth removing this resolver if you can get away with it, due to the security concerns recently christened [`MavenGate`](https://blog.oversecured.com/Introducing-MavenGate-a-supply-chain-attack-method-for-Java-and-Android-applications/)!

```suggestion
```